apps/volunteer/src/pages/shelters/profile/index.tsx

```diff
@@ -1,29 +1,48 @@
 import { Box, Divider } from '@chakra-ui/react';
+import { useQuery } from '@tanstack/react-query';
+import { useParams } from 'react-router-dom';
 import InfoTextList from 'shared/components/InfoTextList';
 import ProfileInfo from 'shared/components/ProfileInfo';
 import Tabs from 'shared/components/Tabs';
 
+import { getShelterProfileDetail } from '@/apis/shelter';
+
 import ShelterRecruitments from './_components/ShelterRecruitments';
-import ShelterReviews from './_components/ShelterReviews';
+import ShelterReviewsTab from './_components/ShelterReviews';
 
 export default function SheltersProfilePage() {
+  const { id } = useParams();
```

Comment:
```suggestion
  const { id } = useParams<{ id: string} >();
```

이런식으로 타입을 지정해주면 좋을 것 같습니다!